makelib/golang.mk

```diff
@@ -251,6 +256,7 @@ go.generate:
 	@$(INFO) go generate $(PLATFORM)
 	@CGO_ENABLED=0 $(GOHOST) generate $(GO_COMMON_FLAGS) $(GO_PACKAGES) $(GO_INTEGRATION_TEST_PACKAGES) || $(FAIL)
 	@$(OK) go generate $(PLATFORM)
+	@$(MAKE) go.imports.fix
```

Comment:
my make-fu is weak, is there a better way to make sure that generated code is properly formatted and `goimports` clean?

Reply:
This seems sensible to me!

Reply:
`go.imports.fix` will run even for non-generated code which seems a bit odd.

Reply:
I do agree that it's odd to have `goimports` run over the entire codebase instead of just the generated code.  I couldn't determine any way to scope `goimports` to only run over generated code (it's not clear exactly what code was generated), so this was the best functional approach I could find.

Any other suggestions for how to ensure arbitrary generated code is always clean?

Reply:
one idea would be to do something like:

```
find $(GO_SUBDIRS) -type f -name 'zz_generated_*' -exec $(GOIMPORTS) ... {} \;
```

Reply:
I could try doing that.  My initial concern is that I wanted this to be generic and be able to catch any generated code, instead of only the `zz_generated*` pattern that controller-tools uses.  That may be fine actually though to scope it to those Kubernetes files.

Reply:
might be a good convention to use ```zz_generated``` for all generated files (kubebuilder or not).

Reply:
I don't feel strongly about this. its just a bit off to run `make generate` and have a bunch of other files also change because there was a surprising `go.imports` run. its minor.

Reply:
It wasn't sitting right with me to run goimports over the entire codebase, so I got it working to use the suggestion of `find zz_generated*` files. PTAL @bassam, thanks!